### PR TITLE
Gameplay/game zone expand

### PR DESCRIPTION
**Pull request by SirZach**
addresses #117 

let me know what you think!

### CODE REVIEW

**Comment by ahaurw01** [comment]
Oh it's beautiful.

File: app/routes/game.js [hunk 1]
```diff
@@ -48,6 +49,21 @@ export default Ember.Route.extend({
 
       // Save the model with the new participant state.
       return model.save();
+    })
+    .then((game) => {
+      //load all the real card models into the store for future use
+      var players = game.get('players');
+      var promises = players.reduce((prev, p) => {
+        prev = prev.concat(p.get('gameCards').map((card) => store.find('card', card.cardId)));
```
Comment (ahaurw01) [inline app/routes/game.js]: When creating a new game, this `p.get` was not a function at times. I used the following workaround, but not sure if this is the best approach.

```javascript
var gameCards = Ember.get(p, 'gameCards') || [];
```

**Comment by ahaurw01** [comment]
Not quite sure why yet, but when expanding zones other than the hand, I don't see the card image (``).

**Comment by SirZach** [comment]
ok awesome feedback. i think i fixed the two issues mentioned..

**Comment by SirZach** [comment]
gonna merge this in. feel free to open up any issues that come about.

**Comment by ahaurw01** [comment]
Nice, good call. Thanks.
I noticed that clicking the context menu on the library causes all the cards to be shown there, as does just clicking anywhere on the library zone. What do you think about only showing the library as a result of clicking some "search library" option in the library context menu? I'll create a ticket and we can continue the discussion there.

**Pull request merged**